rero_ils/modules/acq_order_lines/listener.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+# Copyright (C) 2021 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Signals connector for Acquisition order line."""
+from rero_ils.modules.acq_invoices.api import AcquisitionInvoicesSearch
+from rero_ils.modules.acq_order_lines.api import AcqOrderLine, \
+    AcqOrderLinesSearch
+
+
+def enrich_acq_order_line_data(sender, json=None, record=None, index=None,
+                               doc_type=None, arguments=None, **dummy_kwargs):
+    """Signal sent before a record is indexed.
+
+    :param json: The dumped record dictionary which can be modified.
+    :param record: The record being indexed.
+    :param index: The index in which the record will be indexed.
+    :param doc_type: The doc_type for the record.
```

Comment:
```suggestion
    :param doc_type: The document type of the record.
```
As long as it is documentation, code could be avoided. But maybe `doc_type` is clearer, your call. :slightly_smiling_face: